### PR TITLE
feat(cb2-15352): refactor and add VTP/G 12

### PR DESCRIPTION
**Pull request by naathanbrown**
## Ticket title

_One line description_
[link to ticket number]()

## Checklist

- [ ] Branch is rebased against the latest develop
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo

### CODE REVIEW

**Comment by naathanbrown** [comment]
The snyk issue does not show on npm audit

File: src/functions/EmailRequestProcessor.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { S3Event, S3EventRecord, SQSRecord } from 'aws-lambda';
```
Comment (cb-cs) [inline src/functions/EmailRequestProcessor.ts:1]: Should this live in the services directory? Or maybe even a separate processor directory?

Comment (naathanbrown) [reply src/functions/EmailRequestProcessor.ts:1]: I copied this pattern from cert-gen, what do you think is best? keep them the same or move into a folder?

Comment (cb-cs) [reply src/functions/EmailRequestProcessor.ts:1]: Ah okay fair point, let's leave as-is then if it's consistent. Thanks

**Review by matthew2564 — APPROVED** [review]

**Review by cb-cs — APPROVED** [review]

**Pull request merged**